daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
@alban ack, but `io.EOF` is no "real" error either ;-) Either way, I'm fine with the current solution, and am very much willing to avoid to bikeshed about this :bow: